namer/marathon/src/test/scala/io/buoyant/namer/marathon/MarathonTest.scala

```diff
@@ -38,17 +39,19 @@ class MarathonTest extends FunSuite {
     assert(marathon.uriPrefix.contains("/marathon"))
     assert(marathon._prefix.contains(Path.read("/io.l5d.marathon")))
     assert(marathon.ttlMs.contains(300))
+    assert(marathon.jitterMs.contains(50))
```

Comment:
Would probably also be useful to grab a handful (maybe a few hundred?) of items off the stream and test that they're not all the same, that they all lie between 250 and 350, etc.